### ISSUE TITLE
Feature: Higlight commit author in all repos

### ISSUE THREAD

**Issue by xmedeko** [ForkIssues/TrackerWin#1695]
I've just discovered I can highlight author of commit by choosing a border color around it's name. It's great. However, it's set just for one repository - I have to repeat the setting for every repository.  I would like to set it globally, i.e. set it just once for a certain user and see it in all repositories.

**Comment by bergamin** [comment]
Yes, it's a pain to have to repeat this for every of the 80 repos I manage

**Comment by bergamin** [comment]
Update: _Yes, it's a pain to have to repeat this for every of the 98 repos I manage_ 😭
